files/build_templates/updategraph.service.j2

```diff
@@ -4,7 +4,7 @@ After=rc-local.service
 After=database.service
 Requires=database.service
 {% if sonic_asic_platform == 'mellanox' -%}
-Requires=msn-chassis.service
```

Comment:
why the msn removed? what does it stands for?

Reply:
@liat-grozovik this is the name of service. Aligned according to BSP changes.